platformLibs/src/platform/osx/Intents.def

```diff
@@ -0,0 +1,7 @@
+depends = ApplicationServices CFNetwork ColorSync CoreFoundation CoreGraphics CoreLocation CoreServices CoreText DiskArbitration Foundation IOKit ImageIO Security darwin libkern osx posix
+language = Objective-C
+package = platform.Intents
+modules = Intents
+compilerOpts = -framework Intents
+linkerOpts = -framework Intents
+#Disabled: Unavailable
```

Comment:
Doesn't seem correct.

Reply:
True. `Intents` and `IntentsUI` frameworks are united under SiriKit. I fixed it manually for other OS, but missed it here.